news/2021-05-25-moving-to-libera.md

```diff
@@ -0,0 +1,25 @@
+---
+title: HackSoc's IRC Channels Are Moving To libera.chat
+---
+We're moving!
+
+Due to recent events at Freenode, as summarised [here][summary], 
+HackSoc will be moving all its irc channels from Freenode to 
+[Libera Chat][libera].
+
+The #hacksoc channel on Libera Chat will be bridged to slack in much 
+the same way as the current Freenode channel is. 
+
+#cs-york also has a channel on Libera Chat, with the same people as 
```

Comment:
lolmarkdown

Reply:
fwiw it's Fine when rendered by whatever the actual website uses

Reply:
:thinking: okay then

Reply:
(something something we may want to change markdown parsers in the future so it would be good if this rendered sanely through a commonmark-compatible parser also @ me shut)

Reply:
this is in fact commonmark compliant already :^) commonmark requires a space after the hash(es) for it to be a header, for exactly this reason

Reply:
right, so it's github being good software.

Reply:
question: can we do the thing so I stop getting emails about markdown